test/test_integration.py

```diff
@@ -95,4 +95,5 @@ def test_proximity_estimator_trivial_model(datadir, tmp_path):
     coords = xfms.reference.ndcoords.T
     for i, est in enumerate(em_affines):
         xfm = nt.linear.Affine(xfms.matrix[i], reference=b0nii)
+        est = nt.linear.Affine(est.matrix, reference=b0nii)
```

Comment:
Looks like this is still failing (https://app.circleci.com/pipelines/github/nipreps/eddymotion/286/workflows/7d9d8e2b-f6da-4968-8c21-1b0fbb3775dc/jobs/273). I believe it should be: 
```suggestion
        est = nt.linear.Affine(est, reference=b0nii)
```

Reply:
Yes, `est` is a 4x4 ndarray, not an Affine object.